compiler/tests-spec/testData/diagnostics/notLinked/dfa/pos/52.fir.kt

```diff
@@ -89,7 +89,7 @@ fun <T : String?> T.case_9() = this
 fun case_9() {
     var x: Int? = 10
     x = null
-    <!DEBUG_INFO_EXPRESSION_TYPE("kotlin.Int? & kotlin.Int?")!>x<!>.<!INAPPLICABLE_CANDIDATE!>case_9<!>()
+    <!DEBUG_INFO_EXPRESSION_TYPE("kotlin.Int? & kotlin.Int?")!>x<!>.<!UNSAFE_CALL!>case_9<!>()
```

Comment:
After assigning `null`, it's expected to be resolved to:
```
fun <T: String?> T.case_9() = ...
```
above, but I'd say this looks a lot like https://youtrack.jetbrains.com/issue/KT-41017